src/Components/Organization/Organization.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Divider from '@mui/material/Divider';
+import { ImageListItem, Typography } from '@mui/material';
+import { Card } from './Card';
+
+export default function Organization({
+  name = 'Somos más',
+  image = 'https://picsum.photos/800/350',
+  description = '',
+}) {
```

Comment:
it should receive only the id and then create a getOrganizationData function that return an object with the data of the organization. In the future that getOrganizationData function will be replaced with an endpoint call.

Reply:
so you will need a const [organizationData, setOrganizationData] = useState({
    name: '',
    image: '',
   description: ''
});

and then on useEffect you call the getOrganizationData